bits/sendgrid/README.md

```diff
@@ -0,0 +1,50 @@
+# SendGrid: Email Delivery
+
+This is an Encore package for asynchronous sending emails via [SendGrid](https://sendgrid.com/) using Pub/Sub and the ability to flexibly configure concurrency and retry policies.
+
+## Installation
+
+1. Copy over the `sendgrid` package directory to your Encore application.
+2. Sync your project dependencies by running `go mod tidy`.
+
+## SendGrid API Key
+
+You will need an [API key from SendGrid](https://docs.sendgrid.com/ui/account-and-settings/api-keys) to use this package. You can get one by signing up for a free account at https://sendgrid.com/.
+
+Once you have the API key, set it as an Encore secret using the name `SendGridAPIKey`:
+
+```bash
+$ encore secret set --type dev,prod,local,pr SendGridAPIKey
```

Comment:
Might be worth splitting this into two calls to `encore secret set`, and give guidance to use a separate API key for `prod` compared to the others. Alternatively, have the implementation log a message as opposed to actually sending an email if `encore.Meta().Env.Type != encore.EnvProduction`? Not sure what's best.

Reply:
Thanks, changed readme and added skip sending email in non-prod env
5162819bdc7367db5cd47e03f4706a2f4fed5184